schemas/get-aws-state-response.yml

```diff
@@ -0,0 +1,6 @@
+id:       http://schemas.taskcluster.net/aws-provisioner/v1/get-aws-state-response.json#
+$schema:  http://json-schema.org/draft-04/schema#
+title:                      "Get AWS State Response"
+description: |
+  All of the launch specifications for a worker type
+type:                       object
```

Comment:
I've removed a couple of endpoints which I don't really have a strong reason for continuing to include for now.  I think having a stability keyword is cool, as long as it includes 'deprecated'.  It'd be cool if we automatically marked those as deprecated in the client using https://nodejs.org/api/util.html#util_util_deprecate_function_string